test/models/role_test.rb

```diff
@@ -229,6 +229,10 @@ def test_allow_to_modify_roles_when_bypass_locking
       @role = FactoryGirl.build(:role, :permissions => [])
     end
 
+    it 'should fetch the appropriate permissions' do
+      assert_equal 2, @role.permission_records([[@permission1.name, @permission2.name]]).size
```

Comment:
Since permission_records is a private method, you need to use `send(:permission_records, [[@permission1.anem.....`